manage/upgrading/version_specific_migration/upgrade_to_python3.rst

```diff
@@ -324,24 +343,36 @@ An example for a tox-setup can be found in https://github.com/collective/collect
 Text and Bytes
 ~~~~~~~~~~~~~~
 
-This is by far the biggest issue when porting to Python 3. Read https://portingguide.readthedocs.io/en/latest/strings.html to be prepared.
+This is by far the biggest issue when porting to Python 3.
+Read https://portingguide.readthedocs.io/en/latest/strings.html to be prepared.
 
-As a rule of thumb you can assume that in Python 3 everything should be text and only in very rare cases you need to handle bytes.
+As a rule of thumb, you can assume that in Python 3 everything should be text.
```

Comment:
I would use a "note" directive for the rule of thumb.
This creates a better experience for the user and also makes more clear that it is a "rule of thumb"

Reply:
Fixed in 0fac7f2